pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -296,6 +316,9 @@ int uretprobe__gnutls_record_recv(struct pt_regs *ctx) {
         goto cleanup;
     }
 
+    if (read_len <= 0) {
+        goto cleanup;
+    }
```

Comment:
shouldn't it be at the beginning of the hook? we are doing some irrelevant operations in case of a failed read

Reply:
Fixed